Octokit/Clients/RepositoriesClient.cs

```diff
@@ -502,7 +521,8 @@ public Task<Branch> GetBranch(string owner, string repositoryName, string branch
             Ensure.ArgumentNotNullOrEmptyString(repositoryName, "repositoryName");
             Ensure.ArgumentNotNullOrEmptyString(branchName, "branchName");
 
-            return ApiConnection.Get<Branch>(ApiUrls.RepoBranch(owner, repositoryName, branchName));
+            const string previewAcceptsHeader = "application/vnd.github.loki-preview+json";
```

Comment:
I was following an example I saw in another part of the code base where the preview header was defined each time, but I can change it to a class level. 

Should the unit tests use the same define or their own class level define or specify string  each time?